Platform/CoffeelakeBoardPkg/Script/GpioDataConfig.py

```diff
@@ -0,0 +1,34 @@
+## @ GpioDataConfig.py
+#  This is a Gpio config script for Slim Bootloader
+#
+# Copyright (c) 2020, Intel Corporation. All rights reserved. <BR>
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+##
+
+
+#
+# The index for a group has to match implementation
+# in a platform specific Gpio library.
+#
+

```

Comment:
Having platform name as a mandatory input would be helpful in common script.

Reply:
OK. But how is this input going to be used in the common script?

Reply:
if 'apl', it may return 'unsupported'.